content/ADR-155-scene-triggers-custom-animations.md

```diff
@@ -0,0 +1,73 @@
+- Datapoints and related background information
+- Vocabulary and key terms
+-->
+
+In most experiences in Decentraland you see other players standing still, even if they’re actively engaging with the scene they just stand motionless. This limits a lot of the social aspects of playing with others. It also makes you wonder if other players are even playing at all, or if they’re not even at the keyboard, or if they’re just confused trying to understand the rules. 
+
+While playing alone, these customized animations can also bring a lot more fun into a scene, by making our actions more lively and leaving less to our suspension of disbelief.
+
+## Solution Space Exploration
+
+<!--
+Discuss the potential alternatives and their impact. What alternatives are being considered, their benefits, their costs (team resources, money, time frames), and mitigations for any drawbacks.
+-->
+
+Discuss the potential alternatives and their impact. What alternatives are being considered, their benefits, their costs (team resources, money, time frames), and mitigations for any drawbacks.
+
+## Specification
+
+<!--
+The technical specification should describe the syntax and semantics of any new feature.
+-->
+
+It should be possible for content creators to upload files that include instructions for an avatar animation, following the same limitations as the files uploaded for NFT emotes. These animations however won’t be NFTs, they will be animations that the scene can trigger on any player who visits the scene.
+
+The scene can then trigger these animations on players whenever they make sense in the game’s mechanics. For example, Wondermine could play a “swing” animation whenever a player is mining a meteor, or the casinos could play a “show-cards” animation whenever the player plays their hand in a poker game. A fighting game could include animations for throwing punches, animations for receiving damage, and animations for dying.
+
+As happens currently with all player animations, other players that are around see the animations, the scene creator doesn’t need to do anything extra for this to happen.
+
+As with the triggering of default animations today, the scene should include special permissions to allow it to trigger emotes on an avatar.
+
+Since the animations are downloaded together with the scene's content, there's a corner case that needs to be handled gracefully: Player A sees Player B far away, but has not downloaded the scene where Player B is standing. The scene triggers an animation on Player B, but Player A doesn't have that animation downloaded. In this scenario, it's acceptable for Player A to not see the animation, since Player A is also not seeing the scene that gives that animation context.
+
+
+**Syntax and semantics**
+
+This could be implemented via a component that is added to the player entity, which can be used to trigger animations either from the default list of animations, or from any animation file that is uploaded with the scene. 
```

Comment:
```suggestion
This could be implemented via a component added to the player entity. It could be used to trigger animations from the default list of them or any animation file uploaded with the scene. 
```